src/components/HotspotPairingList.tsx

```diff
@@ -25,6 +25,9 @@ const imagesByHotspotName: Record<string, any> = {
   'SyncroB.it Hotspot': (
     <Image source={require('../assets/images/syncrobit-hotspot.png')} />
   ),
+  'Finestra Miner': (
```

Comment:
Finestra should be below LongAP please. We keep them in the order they were added.